### DISCUSSION TITLE
Playing cards widget?

### DISCUSSION THREAD

**Discussion by m1ga** [elementor/elementor#23137] [Developers Community]
[New: Add the Playing Cards widget](https://github.com/elementor/elementor/pull/23129)

<img src="https://user-images.githubusercontent.com/139438463/255142562-2cb7eb1c-0a73-4948-92ca-51424817ee3c.png"/>

ok :thinking: What would be the use-case of something like this in the core? That looks more like something to have as a ZIP in the developer example docs or an external plugin.

**Comment by hitsgee** [comment]
I thought the same thing. Useless bloat that would be installed. We operate a client base of 700+ active sites and this would not be relevant for use on any.

**Reply by m1ga** [reply]
Since it just displays a card the people who want to use could just use an image/svg. It's not merged yet so let's wait and see.
